spec/api/video_upload_spec.rb

```diff
@@ -60,10 +71,15 @@
                                     'Content-Type': 'application/json'})
     end
 
-    it 'should not be uploaded into S3 bucket' do
-      expect(@response.code).to eq(204)
-      # Check if the picture is not found in S3
-      # to be added
+    # ISSUE: the image is uploaded. The status code should be 400, instead of 204.
+    xit 'returns 400 status code' do
+      expect(@response.code).to eq(400)
+    end
+
+    # ISSUE: the image is uploaded
+    xit 'does not upload the image into s3 bucket' do
```

Comment:
let's use skip: that allows us to also add a comment to be clear what's going on here. 
also rm comment.

```suggestion
  it 'does not upload the image into s3 bucket', skip: 'test fails for now because of reasons' do
```